README.md

```diff
@@ -107,6 +105,15 @@ $http->on('error', function (Exception $e) {
 
 The request object can also emit an error. Checkout [Request](#request) for more details.
 
+When HTTP/1.1 clients want to send a bigger request body, they MAY send only
+the request headers with an additional `Expect: 100-continue` header and
+wait before sending the actual (large) message body.
+In this case the server will automatically send an intermediary
+`HTTP/1.1 100 Continue` response to the client.
+This ensures you will receive the request body without a delay as expected.
+The [Response](#response) still needs to be created as described in the
+examples above.
```

Comment:
:fist_oncoming: The text LGTM, but I still feel that this is the wrong location. Move this up where the request/body is described?